apps/src/code-studio/components/progress/UnitOverviewHeader.jsx

```diff
@@ -20,8 +20,31 @@ import {assignmentCourseVersionShape} from '@cdo/apps/templates/teacherDashboard
 import StudentFeedbackNotification from '@cdo/apps/templates/feedback/StudentFeedbackNotification';
 import VerifiedResourcesNotification from '@cdo/apps/templates/courseOverview/VerifiedResourcesNotification';
 import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import {pegasus, studio} from '../../../lib/util/urlHelpers';
 
 const SCRIPT_OVERVIEW_WIDTH = 1100;
+// Scripts that users will be warned are outdated and have been succeeded by others.
+// This object in the form of {scriptName : description of warning}.
+const OUTDATED_SCRIPT_NAMES_AND_DESC = {
+  course1: i18n.outdatedCourseWarningDescCourses1To4({
+    csFundCourseLink: pegasus('/educate/curriculum/csf-transition-guide')
+  }),
+  course2: i18n.outdatedCourseWarningDescCourses1To4({
+    csFundCourseLink: pegasus('/educate/curriculum/csf-transition-guide')
+  }),
+  course3: i18n.outdatedCourseWarningDescCourses1To4({
+    csFundCourseLink: pegasus('/educate/curriculum/csf-transition-guide')
+  }),
+  course4: i18n.outdatedCourseWarningDescCourses1To4({
+    csFundCourseLink: pegasus('/educate/curriculum/csf-transition-guide')
+  }),
+  '20-hour': i18n.outdatedCourseWarningDescCoursesAccelCourse({
```

Comment:
Minor nit. This name needed to use quotes, it might make sense to just make all of these keys use quotes for consistency.